graphstorm-processing/graphstorm_processing/graph_loaders/dist_heterogeneous_loader.py

```diff
@@ -960,19 +957,30 @@ def _process_node_features(
                 }
                 node_type_feature_metadata[feat_name] = node_feature_metadata_dict
 
+                feat_val = single_feature_df.take(1)[0].asDict().get(feat_name, None)
+                nfeat_size = 1 if isinstance(feat_val, (int, float)) else len(feat_val)
+                ntype_feat_sizes.update({feat_name: nfeat_size})
+
+                # pylint: disable=cell-var-from-loop
```

Comment:
Can we take a second look at https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/cell-var-from-loop.html

Why is the error being raised here, let's ensure we're not falling into the error described in the docs